### PR TITLE
audit: update to 2.8.5.

### PR DESCRIPTION
**Pull request by CameronNemo**

### CODE REVIEW

**Comment by Johnnynator** [comment]
arm* is not compiling. It seems like it is using the wrong python include path when cross compiling. `/usr/include/python2.7` is not prefixed with the cross path.

File: srcpkgs/audit/template [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 pkgname=audit
-version=2.8.4
-revision=2
+version=2.8.5
+revision=1
 build_style=gnu-configure
 configure_args="--libdir=/usr/lib --enable-shared=audit --enable-gssapi-krb5
  --with-apparmor --with-libcap-ng --with-python --with-python3"
-hostmakedepends="automake libtool pkg-config intltool python python3 swig"
+hostmakedepends="automake libtool pkg-config intltool python-devel python3-devel
```
Comment (Johnnynator) [inline srcpkgs/audit/template]: `python*-devel` is most likely not needed.

Comment (CameronNemo) [reply srcpkgs/audit/template]: Yeah, left over from a half assed attempt to fix ARM, then I forgot about it.

**Comment by maxice8** [comment]
Seems to be broken on arm

**Comment by CameronNemo** [comment]
I disabled the Python2 bindings.

**Pull request merged**